igv_remote/_core.py

```diff
@@ -9,19 +9,33 @@
 
 def _append_id(filename, id):
     return "{0}_{2}.{1}".format(*filename.rsplit('.', 1) + [id])
-        
+
+def _parse_loc(chromosome, pos1, pos2=None, expand=20):
+    if expand < 20:
+        print("IGV expands left and right margin by at least 20bp")
+        expand=20
+    start_pos = int(pos1-expand)
+    if pos2 is None:
+        end_pos = int(pos1+expand)
+    else:
+        end_pos = int(pos2)
+    start_pos ='{:,}'.format(int(start_pos))
+    end_pos = '{:,}'.format(int(end_pos))
```

Comment:
Don't need `int` inside the format string. Also, unless you're planning on using this function to print the region elsewhere, you don't need the comma formatting either.

Reply:
Thanks! I tested and realized that IGV itself handles all these different types of input ;) so I removed `int` and `"chr"` and let feed it directly